mlflow/tracking/client.py

```diff
@@ -426,6 +426,30 @@ def create_trace(
             tags=tags,
         )
 
+    def delete_traces(
+        self,
+        experiment_id: str,
+        max_timestamp_millis: Optional[int] = None,
+        max_traces: Optional[int] = None,
+        request_ids: Optional[List[str]] = None,
+    ) -> None:
+        """
+        Delete traces based on the specified criteria.
+
+        Args:
+            experiment_id: ID of the associated experiment.
+            max_timestamp_millis: The maximum timestamp in milliseconds since the UNIX epoch for
+                deleting traces.
+            max_traces: The maximum number of traces to delete.
+            request_ids: A set of request IDs to delete.
```

Comment:
Can we indicate that either `max_timestamp_millis` or `request_ids` must be specified, but not both? Can we also indicate that `max_traces` can't be specified if `request_ids` is specified?

Reply:
/ai Update the docstring based on the feedback.

Reply:
/ai Let's mention these requirements in the function description, not parameter description.